README.md

```diff
@@ -223,25 +222,22 @@ Ember Data attributes [support a `defaultValue` config option](http://emberjs.co
 
 ```javascript
 // app/models/person.js
-import Model from 'ember-data/model';
-import {
-  fragment,
-  fragmentArray,
-  array
-} from 'ember-data-model-fragments/attributes';
 
-export default Model.extend({
-  name      : fragment('name', { defaultValue: { first: 'Faceless', last: 'Man' } }),
-  addresses : fragmentArray('address'),
-  titles    : array('string')
-});
+import Model, { attr, belongsTo, hasMany } from '@ember-data/model';
+import { fragment, fragmentArray, array } from 'ember-data-model-fragments/attributes';
+
+export default class PersonModel extends Model {
+  @fragment('name', { defaultValue: { first: 'Faceless', last: 'Man' } }) name;
+  @fragmentArray('address') addresses;
+  @array('string') titles;
+}
 ```
 
 Since JavaScript objects and arrays are passed by reference, the value of `defaultValue` is copied using `Ember.copy` in order to prevent all instances sharing the same value. If a `defaultValue` option is not specified, `fragment` properties default to `null` and `fragmentArray` properties default to an empty array. Note that this may cause confusion when creating a record with a `fragmentArray` property:
 
 ```javascript
 let person = store.createRecord('person');
-let addresses = person.get('addresses'); // null
+let addresses = person.addresses; // null
```

Comment:
I think `get` is basically still required on models, since attrs can be relationships.